### PR TITLE
Add item bug issue

### LINKED ISSUE #37: Decide on time frame logic and remove inconsistency
Currently we have two contradicting time frame statements, because of inconsistent issue descriptions.

We have 'kind of soon' equal to 14 and 'not soon' equal to 30 in `<AddItem />` ,
while 'kind of soon' is set to 7-30 days in `<List />`. We need to decide the numbers for each time frame and change one of the components correspondingly.

Screenshot of inconsistent time frames in Add Item component and List component

### PR DESCRIPTION
**Pull request by mira-kine**
## Description

This PR fixes bug of inconsistent date/time descriptions in `Add Item` vs. `List`. I have set it to show the range of dates that Soon/Kind of Soon/Not Soon correlates to. To keep it from being too wordy (for ex writing "Between 7 and 30 days" I wrote (7 ~ 30days). I am open to feedback if majority would rather want the wording to be exactly the same as `List` /if we want to change the description in `List` to have less words as well. I also added some light css for readability, but will be fully styling it in another issue.

## Related Issue

Closes #37 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
| ✓     | :bug: Bug fix              |
|   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

### Before
Screen Shot 2022-08-24 at 12 04 45 PM

### After

Screen Shot 2022-08-24 at 12 04 00 PM

## Testing Steps / QA Criteria

1. From your local machine git checkout main then git pull
git checkout add-item-bug-issue
2. npm run start
3. Go to "Add Item" and compare date consistency with "List"

### CODE REVIEW

File: src/views/AddItem.jsx [hunk 1]
```diff
@@ -85,7 +86,7 @@ export function AddItem({ listToken, itemList, setData }) {
 								value="14"
 								onChange={handleTime}
 							/>
-							Kind of Soon (14 days)
+							Kind of Soon (7 ~ 30 days)
```
Comment (N-Zubko) [inline src/views/AddItem.jsx:89]: It bothers me that in the first case we use the max value for the calculation, in the third case it is the min value (30), and in the second case it is 14, which is closer to the the min value to 7.

May be it would be easier to change only the description in the list view to _soon 7 days, kind of soon 7-14, not so/that soon 14-30 and inactive over 30_  so that only max values are used in the calculation?

Comment (N-Zubko) [reply src/views/AddItem.jsx:89]: I mean that 60 days out of nowhere in one of the last tickets was really confusing, because we used only 7, 14, 30 before

File: src/views/AddItem.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.date-container {
+	display: flex;
+	flex-direction: column;
+}
+
+.timeframe {
```
Comment (N-Zubko) [inline src/views/AddItem.css:6]: with flex it looks much nicer and readable!

File: src/views/AddItem.jsx (hunk 1 above)
Comment (Trevor-Rezac) [inline src/views/AddItem.jsx:89]: I am ok with the days as they currently are. I might add a suggestion though. Doing something like Soon (~7 days) Kind of soon (~30 days) Not that soon (30+ days). It saves character space for one.  And we could then use the same descriptions on the List page and have Inactive show as Inactive (60+ days). Thoughts?

**Review by Trevor-Rezac — APPROVED** [review]
I think this looks great! I made one minor suggestion, but I am fine with things either way so I'm going to go ahead and approve this PR!

File: src/views/AddItem.jsx (hunk 1 above)
Comment (michellerenehey) [reply src/views/AddItem.jsx:89]: My preference would be to keep the days as they currently are, but alter them _ever so slightly_:

Comment (michellerenehey) [reply src/views/AddItem.jsx:89]: We could be a little less specific? It's wordier, but makes a little more sense than specific days? 

Soon: within the week
Kind of Soon: within the month
Not That Soon: in more than a month

Soon: this week
Kind of Soon: this month
Not That Soon: next month or beyond

Otherwise, I'd only prefer that a number that ends one timeframe doesn't start another! If I have an item that I want to buy in 7 days, do I put it in Soon or Kind of Soon?! 😆

File: src/views/AddItem.jsx [hunk 3]
```diff
@@ -96,7 +97,7 @@ export function AddItem({ listToken, itemList, setData }) {
 								value="30"
 								onChange={handleTime}
 							/>
-							Not Soon (30 days)
+							Next month
```
Comment (N-Zubko) [inline src/views/AddItem.jsx:100]: May be it is better to put more general 'later' as @michellerenehey suggested on Slack?

**Review by N-Zubko — APPROVED** [review]
I am approving it, I am glad we reached an understanding on values. There is one suggestion about the name of the third category (15-30 days).

File: src/views/AddItem.jsx (hunk 3 above)
Comment (michellerenehey) [reply src/views/AddItem.jsx:100]: I think 'next month' is fine! I like that it all fits together: this week, this month, next month. Feels clean. 

 I was thinking more about the dilemma and came to a brain conclusion:  If a user puts 'next month' and then the month passes (or, well, 30+ days pass... 45 days pass....60 days pass), then it's no longer 'next month' and it will make sense for it to be inactive!

**Review by michellerenehey — APPROVED** [review]
This was a logical doozy! Great job, Mira! Approved. 🙌

**Review by lindseyindev — APPROVED** [review]
I saw a lot of the slack discussion and there's been great collaboration here on bringing more cohesion and simplicity to the UX, I like the balancing you've done of trending towards being less specific on the numbers but still referencing general date and time frames direction you've gone in to give a sense still of the organization of the list but make it easier for users!

**Review by nickytonline — APPROVED** [review]
Great collaboration from the office hours, to the Slack convos. This looks great!
